solver/src/main/java/org/chocosolver/solver/variables/view/IView.java

```diff
@@ -38,7 +37,7 @@
      *
      * @return variable observed
      */
-    IntVar getVariable();
+    Variable getVariable();
```

Comment:
We can consider generifying the class, don't you think ? Well, it can be declared in a distinct issue, since that's not central to this PR

Reply:
I totally agree, I will do a small PR for that.